src/validators/bartio/defaultValidatorList.json

```diff
@@ -1,6 +1,14 @@
 {
   "name": "Berachain Bartio Validator List",
   "validators": [
+    {
+      "id": "0xe868bE65C50b61E81A3fC5cB5A7916090B05eb2A",
+      "logoURI": "https://doubletop-bin.ams3.digitaloceanspaces.com/2top.png",
```

Comment:
Please upload your logo to `/src/assets` and prefix the `logoURI` with cloudinary link